xml/core_registers.xml

```diff
@@ -71,7 +71,10 @@
 
             Other values are reserved for future use.
         </field>
-        <field name="0" bits="5:3" access="R" reset="0" />
+        <field name="0" bits="5:4" access="R" reset="0" />
+        <field name="nmip" bits="3" access="R" reset="0">
+            When set, there is a Non-Maskable-Interrupt (NMI) pending for the hart.
```

Comment:
Can you add something like:
Since NMIs indicate a hardware error condition, reliable debugging may no longer be possible once this bit becomes set.